_posts/2020-01-06-2019-reading-in-review.md

```diff
@@ -0,0 +1,501 @@
+---
+layout: post
+title: "2019 Reading in Review"
+date: "2020-01-06 12:00:00 -0600"
+---
+
+I used to review books on [a dedicated Mastodon
+account](https://wandering.shop/@benhamill), but I've since wanted to be able to
+show them to people outside the context of the fediverse. This was also spurred
+by the fact that, after moving, I read a ton of books without keeping up with
+the reviews, so I'd intended to do bunch of catch-up reviews… and that was in
+October 2018.
+
+So what's _actually_ transpired is that I've kept a one-line entry for every
+book I've read including the title, author and finish date except for the stuff
+I read right at the end of 2018 and the start of 2019. So I'm going to share
+that list here and maybe talk about some of them a bit and see how I feel.
+
+Also, I want to point out that this list doesn't include comics, which I mostly
+read in trade paperback form, or stuff I read with my kids, which is mostly very
+short but has begun to include some chapter books. Many of those things have
+been lovely and totally worth recommending or talking about, but I figure I have
+to limit scope somehow and I didn't happen to think to put those into my
+journal.
+
+
+## The Numbers
+
+Let's do some stats, shall we? Obviously I have to excluse some stuff from the
+duration stats on the basis of not knowing when I read it.
+
+**Number of Things Read:** 47
+
+**Reading Time** (including weekends, during which I generally don't read):
+* Maximum: 21 days (_Jacobin 33_ and _The Prey of Gods_)
+* [Median](https://en.wikipedia.org/wiki/Median): 8 days (_A Conjuring of Light_ and _Jacobin 34_)
+* Minimum: 0 days (_Artificial Condition_, _Exit Strategy_, _Capitalism & the
```

Comment:
0 days implies that you didn't read these to me. If you're indicating less than a day, something other than zero seems like a better path.